### ISSUE TITLE
flatpak install --deps-only

### ISSUE THREAD

**Issue by damianatorrpm** [flatpak/flatpak#3047]
```
flatpak install --deps-only 
flatpak --user install 
```

Save space.

**Comment by aiamuzz** [comment]
> ```
> flatpak install --deps-only #/var/lib/flatpak
> flatpak --user install  #$HOME/.local/share/flatpak
> ```
> 
> Save time & space.

hmmmm ...

will these commands work like this ?

```
flatpak install --deps-only #/media/username/externalPartition/flatpak
flatpak --user install #/media/username/externalPartition/flatpak
```

If they work then i guess we will save having to setup the extras.conf file and save having to type `--installation=myflatpakcollection`

just installing with `--user` will be sufficient to install at the location set ...

**Comment by damianatorrpm** [comment]
No, there is no --deps-only that is the point of the report #is comment to make it more clear that you are using system wide installation for runtimes to save space.

**Comment by bochecha** [comment]
Note that if this is implemented it will lead to #2842.
